src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is doing its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck has stopped working.");

```

Comment:
Similarly, the message 'Truck has stopped working.' could be more descriptive. Consider specifying what indicates that the Truck has stopped its work.